src/pages/CommitDetailPage/CommitCoverage/UploadsCard/UploadItem.test.tsx

```diff
@@ -152,19 +200,57 @@ describe('UploadsCard', () => {
   })
 
   describe('rendering errors', () => {
-    let consoleSpy
-    beforeAll(() => {
-      consoleSpy = vi.spyOn(console, 'error').mockImplementation(() => {})
+    beforeEach(() => {
+      // Suppress prop-type warnings.
+      vi.spyOn(console, 'error').mockImplementation(() => {})
```

Comment:
No cleanup? 👀

Reply:
LOL I did this at the same time I DM'd you. Never came back to it I guess.

Reply:
Turns out no errors are even logged out! Removed the whole thing :)